pkg/generate/app/componentresolvers.go

```diff
@@ -299,7 +300,11 @@ func resolveExact(resolver Resolver, value string) (exact *ComponentMatch, inexa
 			return nil, nil, err
 		}
 	}
-	return match, nil, nil
+	if match.Score == 0.0 {
+		return match, nil, nil
+	} else {
```

Comment:
So an inexact match, of which I only find one, we are going to give you a message saying "hey, we only found jenkins-1 for jenkins, tell us you want to use jenkins-1" sounds irritating to end users.